quipucords/quipucords/settings.py

```diff
@@ -131,17 +131,37 @@
 # Database
 # https://docs.djangoproject.com/en/1.11/ref/settings/#databases
 
-DEV_DB = os.path.join(BASE_DIR, 'db.sqlite3')
-PROD_DB = os.path.join(os.environ.get('DJANGO_DB_PATH', BASE_DIR),
-                       'db.sqlite3')
-DB_PATH = PROD_DB if PRODUCTION else DEV_DB
-
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': DB_PATH,
+# Database Management System could be 'sqlite3' or 'postgresql'
+DBMS = os.getenv('DBMS', 'sqlite3').lower()
+
+if DBMS == 'sqlite3':
+    DEV_DB = os.path.join(BASE_DIR, 'db.sqlite3')
+    PROD_DB = os.path.join(os.environ.get('DJANGO_DB_PATH', BASE_DIR),
+                           'db.sqlite3')
+    DB_PATH = PROD_DB if PRODUCTION else DEV_DB
+    DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.sqlite3',
+            'NAME': DB_PATH,
+        }
+    }
+
+elif DBMS == 'postgresql':
+    PGDATABASE = os.getenv('PGDATABASE')
```

Comment:
Maybe you should try direct key access here, because if any of these env vars is not defined then it won't work.

If you would like to have some default values maybe these would work better:

```
PGDATABASE = postgres
PGHOST = localhost or ::
PGUSER = postgres
PGPASSWORD = postgres
PGPORT = 5432
```

By work better I mean some of the default values we find out there.